llvm/utils/TableGen/IntrinsicEmitter.cpp

```diff
@@ -229,8 +229,9 @@ struct IntrinsicTargetInfo {
 };
 static constexpr IntrinsicTargetInfo TargetInfos[] = {
 )";
-  for (const auto [Name, Offset, Count] : Ints.Targets)
-    OS << formatv("  {{\"{}\", {}, {}},\n", Name, Offset, Count);
+  for (const auto &Target : Ints.Targets)
```

Comment:
I recall references is not good for small value, maybe ignore them and just remove the `const`?

Reply:
Done, removed reference and `const`.